### ISSUE TITLE
Pages lack meaningful page titles

### ISSUE THREAD

**Issue by weboverhauls** [Polymer/shop#40]
All page titles are the same, "SHOP". Not beneficial for accessibility (nor usability).

Suggest prepending page name. Examples:
Men's T-Shirts - SHOP
Ladies Outerwear - SHOP
Your Cart - SHOP

WCAG Reference:
https://www.w3.org/TR/2008/REC-WCAG20-20081211/#navigation-mechanisms-title

**Comment by frankiefu** [comment]
This is already fixed in https://github.com/Polymer/shop/pull/32
We will try to push a new build to shop.polymer-project.org either later today and early next week.
